src/hogben/magnetism.py

```diff
@@ -1,19 +1,23 @@
 import matplotlib.pyplot as plt
-plt.rcParams['figure.figsize'] = (9,7)
-plt.rcParams['figure.dpi'] = 600
+import os
+import sys
 
 import numpy as np
-import os, sys
+
+from scipy.optimize import differential_evolution
+
+from hogben.models.magnetic import SampleYIG
+from hogben.simulate import simulate_magnetic, reflectivity
+from hogben.utils import save_plot
+
 # Add the models directory to the system path.
 # Add the current directory to the path to avoid issues with threading.
 sys.path.append(os.path.join(os.path.dirname(__file__), 'models'))
```

Comment:
I think that this can be achieved with something like 

```
from ..models.magnetic import SamplYIG
```

This could remove the need to import `os` and `sys` (which I am not a massive fan of using if possible).

Reply:
totally agree with the sys, os, and python path, but thought I'd try to fix one thing at a time, since I wasn't sure if this would break stuff by itself. The sys.path.append statements are next to go